Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/ui/overview/adapter/ContactDiaryOverviewNestedAdapter.kt

```diff
@@ -0,0 +1,37 @@
+package de.rki.coronawarnapp.contactdiary.ui.overview.adapter
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import de.rki.coronawarnapp.databinding.IncludeContactDiaryOverviewNestedItemBinding
+
+class ContactDiaryOverviewNestedAdapter :
+    RecyclerView.Adapter<ContactDiaryOverviewNestedAdapter.NestedItemViewHolder>() {
+
+    private val dataList: MutableList<ListItem.Data> = mutableListOf()
```

Comment:
here, you use a `val` instead of a var, clear it if new data is set and add the new data. Above, in the overview adapter you do use a var and reset the list. Align it please.

Reply:
true.
adjusted